src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,163 @@
+
+Since the launch of the staking deposit contract in November 2020, some brave Ethereum pioneers have voluntarily locked funds up to activate "validators", the accounts that have the right to formally attest to and propose blocks, following network rules.
+
+Before the Shanghai update, you couldn't use or get your money out if you staked ETH. But now, you can opt-in to automatically receive your rewards into a provided account, and you can also take out all your money whenever you want.
+
+### How do I prepare? {#how-do-i-prepare}
+
+<WithdrawalsTabComparison />
+
+### Important notices {#important-notices}
+
+Providing a withdrawal address is a required step for any validator account before it will be eligible to have ETH withdrawn from its balance.
+
+<InfoBanner emoji="⚠️" isWarning>
+  <strong>Each validator account can only be assigned a single withdrawal address, one time.</strong> Once an address is chosen and submitted to the Beacon Chain, this cannot be undone or changed again. Double-check ownership and accuracy of the address provided before submitting.
+</InfoBanner>
+
+There is <strong>no threat to your funds in the meantime</strong> for not providing this. Failure to add withdrawal credentials will simply leave the ETH locked in the validator account as it has been until a withdrawal address is provided.
+
+## Exiting staking entirely {#exiting-staking-entirely}
+
+Providing a withdrawal address is required before _any_ funds can be transferred out of a validator account balance.
+
+Users looking to exit staking entirely and withdraw their full balance back must also sign and broadcast a "voluntary exit" message with validator keys which will start the process of exiting from staking. This is done with your validator client and submitted to your beacon node, and does not require gas.
+
+The process of a validator exiting from staking takes variable amounts of time, depending on how many others are exiting at the same time. Once complete, this account will no longer be responsible for performing validator network duties, is no longer eligible for rewards, and no longer has their ETH "at stake". At this time the account with be marked as fully “withdrawable”.
+
+Once an account is flagged as "withdrawable", and withdrawal credentials have been provided, there is nothing more a user needs to do aside from wait. Accounts are automatically and continuously swept by block proposers for eligible exited funds, and your account balance will be transferred in full (also known as a "full withdrawal") during the next sweep (more on this [below](#validator-sweeping)).
+
+## When are staking withdrawals enabled? {#when}
+
+Withdrawal functionality will be enabled through a two-part simultaneous network upgrade, **Shanghai + Capella**.
+
+<ShanghaiCapella />
```

Comment:
Wondering if somehow we could write the content in the md file and accept it as its children 🤔 for translation concerns.

Reply:
Same as above, I agree. Let's scope that to a subsequent initiative if you're okay with that.